src/canvas.class.js

```diff
@@ -339,7 +339,7 @@
 
     /**
      * hold the list of nested targets hovered
-     * @type fabric.Object
+     * @type Array[fabric.Object]
```

Comment:
my linter was complaining that i couldn't set this._hoveredTargets to an array

Reply:
JSDOCS has to be as much good as possible, hoping one day we can pull a ts definition directly from them.